nginx/includes/redirects.conf

```
@@ -176,7 +176,8 @@ rewrite ^/docs/category/build-applications$ "/docs/build/apps" permanent;
 rewrite ^/docs/building-apps(.*)$ "/docs/build/apps$1" permanent;
 rewrite ^/docs/smart-contracts$ "/docs/build/smart-contracts/overview" permanent;
 rewrite ^/docs/smart-contracts/guides(.*)$ "/docs/build/guides$1" permanent;
-rewrite ^/docs/smart-contracts/smart-wallets(.*)$ "/docs/build/apps/smart-wallets$1" permanent;
+rewrite ^/docs/smart-contracts/smart-wallets(.*)$ "/docs/build/guides/contract-accounts/smart-wallets/$1" permanent;
+rewrite ^/docs/build/apps/smart-wallets(.*)$ "/docs/build/guides/contract-accounts/smart-wallets/$1" permanent;
```

```suggestion
rewrite ^/docs/smart-contracts/smart-wallets(.*)$ "/docs/build/guides/contract-accounts$1" permanent;
rewrite ^/docs/build/apps/smart-wallets(.*)$ "/docs/build/guides/contract-accounts$1" permanent;
```

we'll probably want these pointing to the "main" how-to guides page for contract accounts? or, maybe we should send them to one of the sub-pages? in any case, we'll definitely want to drop the `/` before the capture group (`$1`) gets put on the end

---

Committed as is. Sending them to the main page seems sensible to me since the subpages are easily discoverable from there.